person.rb

```diff
@@ -7,17 +9,25 @@ def initialize(age, name = 'unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @corrector = Corrector.new
+  end
+
+  def can_use_services?
+    of_age? || parent_permission
+  end
+
+  def validate_name
+    @name = @corrector.correct_name(name)
```

Comment:
- [ ] In order to be sure this method works correctly you should pass the instance variable to the correct_name method.